### ISSUE TITLE
Predict using only keras functions?

### ISSUE THREAD

**Issue by edofazza** [ultralytics/yolov5#7841]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I generated the yolov5s_saved_model modifying one line of code in export.py in order to obtain the keras_metadata.pb inside it and making the directory loadable from keras.models.load_model(path). Anyway, there is now way after that to predict correctly bounding boxes since it seems that the predictions are completely random and it ends up considering everything a 'dining table' or a 'traffic light', also the bounding boxes are not correctly located. I checked the weights if they were correct and they are, so everything should be fine. Have somebody managed to make it work?

I cannot use the predict.py file, since it uses pytorch for predicting and I need to have all the code using the Tensorflow library, since I am going to use a different real-value representation backend which is not implemented in pytorch.

**Comment by waynetee** [comment]
Did you normalize the image you are inputting? I.e. `image = image/255`

**Comment by edofazza** [comment]
That was it! Thank you

**Comment by glenn-jocher** [comment]
@edofazza you're welcome! Glad to hear that it resolved the issue. If you have any more questions or need further assistance, feel free to ask. Good luck with your project!
